search/django/weedid/notification.py

```diff
@@ -0,0 +1,34 @@
+from core.settings import SITE_BASE_URL
+from weedid.models import Dataset, WeedidUser
+from weedid.utils import send_email
+
+
+def upload_notification(upload_id):
+    upload_entity = Dataset.objects.get(upload_id=upload_id)
+    uploader = upload_entity.user
+    email_body = f"""\
+    User {uploader.username} <{uploader.email}> has uploaded a dataset {upload_entity.metadata['name']} to review.
+    {'You can check it from ' + SITE_BASE_URL + '/datasets/' + upload_id}.
+
+    """
+    staff_recipients = [
+        staff.email for staff in WeedidUser.objects.filter(is_staff=True)
+    ]
+    send_email(
+        f"New upload ({upload_entity.metadata['name']})", email_body, staff_recipients
+    )
+
+
+def review_notification(message, upload_id):
+    upload_entity = Dataset.objects.get(upload_id=upload_id)
+    uploader = upload_entity.user
+    email_body = f"""\
+    Your dataset upload {upload_entity.metadata['name']} has been {message} after review
```

Comment:
Please make this a bit more formal, e.g. start with "Dear Weed-AI contributor,"

Best would be that we have this template in an external file, that can be templated by jinja. This would make it easier for @geezacoleman or other to edit how the email appears to the contributor.